cmd/sops/main.go

```diff
@@ -960,13 +960,22 @@ func keyGroups(c *cli.Context, file string) ([]sops.KeyGroup, error) {
 	return []sops.KeyGroup{group}, nil
 }
 
+// getConfigPath returns the config path that should be used, either by finding it
```

Comment:
nit: the current phrasing is a bit confusing and could lead people to believe that it will first find from a given root, and it that fails it will use the one passed through the command line, when it's actually the opposite